lib/src/conflicts.rs

```diff
@@ -367,10 +367,12 @@ pub fn materialized_diff_stream<'a>(
         .buffered((store.concurrency() / 2).max(1))
 }
 
-/// Parses conflict markers from a slice. Returns None if there were no valid
-/// conflict markers. The caller has to provide the expected number of merge
-/// sides (adds). Conflict markers that are otherwise valid will be considered
-/// invalid if they don't have the expected arity.
+/// Parse conflict markers from a slice
```

Comment:
nit: "Parse**s**" and terminate with period? I think that's the convention of Rust std doc.

(applies to other doc changes)